### ISSUE TITLE
amp-bind doesn't change img src

### ISSUE THREAD

**Issue by cramforce** [ampproject/amphtml#7296]
Issue first seen here https://github.com/ampproject/amp-by-example/issues/586

It appears that changing the `src` of an `amp-img` does not update the `src` of the `img`.

**Comment by choumx** [comment]
Most likely due to stale AMP runtime in SW cache. E.g. My cached version was 1482359659944 which lacked the `mutatedAttributesCallback` method. Can't repro after clearing cache.

Please reopen if still reproducible after clearing cache.

/cc @sebastianbenz Perhaps amp-by-example should change [caching strategy](https://googlechrome.github.io/sw-toolbox/docs/master/tutorial-api) from `cacheFirst` to `fastest` to avoid these types of problems for new/experimental components?

**Comment by sebastianbenz** [comment]
I can still reproduce it via this page
https://callous-cheetah.gomix.me/amp-bind-xhr.html (with dev-channel only).

**Comment by cramforce** [comment]
This still happens for me. There is an error in the console:

```
cdn.ampproject.org/v0/amp-bind-0.1.js:16 Uncaught (in promise) TypeError: Cannot read property 'enqueue' of undefined _reported_
    at lb (cdn.ampproject.org/v0/amp-bind-0.1.js:16)
    at jb (cdn.ampproject.org/v0/amp-bind-0.1.js:14)
    at cdn.ampproject.org/v0/amp-bind-0.1.js:127
    at Promise (<anonymous>)
    at Tb (cdn.ampproject.org/v0/amp-bind-0.1.js:126)
    at Rb (cdn.ampproject.org/v0/amp-bind-0.1.js:124)
    at cdn.ampproject.org/v0/amp-bind-0.1.js:123
    at <anonymous>
```

**Comment by choumx** [comment]
Ah, that's actually the chunk-related bug in 1486166232176 that was fixed in #7374. I'll re-close when I verify the fix is live on this week's canary.

**Comment by choumx** [comment]
Verified fixed in new canary 1486754279878.
